src/main/cli.ts

```diff
@@ -0,0 +1,98 @@
+import { KANGAROO_CONFIG } from './const';
+import { breakingAppVersion } from './filesystem';
+import { app } from 'electron';
+
+export interface CliOpts {
+  profile?: string;
+  networkSeed?: string | undefined;
```

Comment:
Is it necessary to define `string | undefined` with a `?` field?

Reply:
no, of course not. Will remove it.